css-canvas-medical/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="row justify-between">
+      <img src="images/logo-blue.svg" alt="Canvas logo">
+      <nav class="row">
+        <a href="#product" class="nav-item">Product</a>
+        <a href="#implementation" class= "nav-item">Implementation</a>
+        <a href="#about-us" class="nav-item">About us</a>
+        <a href="#blog" class="nav-item">Blog</a>
+        <button class="blue-button">Contact Us</button>
+      </nav>
+    </header>
+    <div class="row hero-image">
+      <img src="images/hero.jpg">
+      <div class="hero-content relative">
+        <h1>The EMR Reimagined for Primary Care</h1>
+        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eveniet magni assumenda, dolores iure ipsam commodi hic, mollitia deleniti illo voluptatibus minima omnis facilis tenetur officiis fugiat doloremque nisi maiores minus accusantium ipsum ducimus? Ducimus, ex deleniti tenetur dicta perspiciatis fugiat ipsa quam magni expedita, sapiente omnis! Sit doloribus consequuntur labore!</p>
+        <button class="blue-button">Get Started</button>
+      </div>
+    </div>
+    <div class="row places">
+      <div class="column-full">
+        <h2>Clinicians Prefer Canvas</h2>
+        <p>Physician-owned primary care practices across the country have<br />already made Canvas the fabric of their care.</p>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <p>Cascade Family Pracitce, Portland OR</p>
+          <p>Temescal Cree Medicine, Oakland CA</p>
+          <p>San Juan Healthcare, Friday Harbor WA</p>
+        </div>
+        <div class="column-one-half">
+          <p>Ward Medical Clinic, Bonifay FL</p>
+          <p>Caribou Medical, Towanda PA</p>
+          <p>Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
+    </div>
+    <div class="row learn">
+      <div class="column-full justify-between">
+        <h2>Ready to learn more?</h2>
+        <p>Schedule a private meeting now</p>
+      </div>
+        <button class="blue-button">Get Started</button>
+    </div>
+    <div class="row bottom">
+      <div class="column-one-half first">
+        <img src="images/logo-white.png" alt="Canvas logo">
+        <p>2307 Irving Street, Suite 228<br />San Franscisco CA 942122<br />(800) 370-1416<br /> info@canvasmedical.com</p>
+      </div>
+      <div class="column-one-half row">
```

Comment:
No one element should be a `column` and a `row`. You should make a `row` element as the child of this `column` element